### PR TITLE
allow reason phrase to be empty

### PR DESCRIPTION
**Pull request by dontub**

### CODE REVIEW

**Comment by Nyholm** [comment]
Sorry for the late response. Could you give me a reason you you need this?

**Comment by dontub** [comment]
The [Apple Push Notification service](https://developer.apple.com/library/content/documentation/NetworkingInternet/Conceptual/RemoteNotificationsPG/APNSOverview.html) just returns a status code, but no reason phrase.

**Comment by Nyholm** [comment]
Thanks. Would you mind adding a small test for this?

File: test/Buzz/Test/Message/ResponseTest.php [hunk 1]
```diff
@@ -50,6 +50,15 @@ public function testGetReasonPhraseReturnsAMultiwordReasonPhrase()
         $this->assertEquals('Internal Server Error', $response->getReasonPhrase());
     }
 
+    public function testGetReasonPhraseReturnsEmptyReasonPhrase()
+    {
+        $response = new Response();
+
+        $response->addHeader('HTTP/1.0 200');
+
+        $this->assertNull($response->getReasonPhrase());
```
Comment (Nyholm) [inline test/Buzz/Test/Message/ResponseTest.php]: Good. Could you also assert status code and protocol here?

**Comment by Nyholm** [comment]
Thank you.

**Comment by artursvonda** [comment]
@Nyholm Any plans to release this soon? We've run into an issue where we needs this.

**Comment by Nyholm** [comment]
Yes. 
I'm on vacation now. I'll be back in the end of the month. I will prioritize Buzz for the rest of the year.

**Comment by artursvonda** [comment]
Thanks! Enjoy the vacation!

**Pull request merged**